generated/csharp/GitHub/Middleware/Options/UserAgentOptions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Kiota.Abstractions;
+
+namespace GitHub.Octokit.Client.Middleware.Options;
+
+public class UserAgentOptions : IRequestOption
+{
+    public string ProductName { get; set; } = "octokit-net";
```

Comment:
If we're sticking with the go-sdk pattern, perhaps this should be "dotnet-sdk".

Reply:
done!